pkg/operator/podsecurityreadinesscontroller/violation.go

```diff
@@ -54,6 +50,28 @@ func (c *PodSecurityReadinessController) isNamespaceViolating(ctx context.Contex
 	return len(c.warningsHandler.PopAll()) > 0, nil
 }
 
+func determineEnforceLabelForNamespace(ns *applyconfiguration.NamespaceApplyConfiguration) (string, error) {
+	if _, ok := ns.Annotations[securityv1.MinimallySufficientPodSecurityStandard]; ok {
+		// Pick the MinimallySufficientPodSecurityStandard if it exists
```

Comment:
Do you think it would be better to phrase the reasoning, that we discussed in chat? Something in the sense like "should be the go-to route, but we keep the older one for upgrade scenarios as of now / other approaches can be removed once annotation is there".